src/App.tsx

```diff
@@ -1,9 +1,18 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.scss';
 import { MoviesList } from './components/MoviesList';
 import moviesFromServer from './api/movies.json';
 
 export const App: React.FC = () => {
+  const [query, setQuery] = useState('');
+  const movieQuery = (movieProps: string) => {
+    return movieProps.toLowerCase().includes(query.toLowerCase().trim());
+  };
+
+  const visibleMovies = moviesFromServer.filter(movie => (
+    movieQuery(movie.title) || movieQuery(movie.description)
```

Comment:
It fucntion returns boolean type, so, rename it
```suggestion
    isMovieIncludes(movie.title) || isMovieIncludes(movie.description)
```